subsys/bluetooth/host/att.c

```diff
@@ -2440,7 +2440,8 @@ static int bt_att_recv(struct bt_l2cap_chan *chan, struct net_buf *buf)
 
 	if (!handler) {
 		BT_WARN("Unhandled ATT code 0x%02x", hdr->code);
-		if (att_op_get_type(hdr->code) != ATT_COMMAND) {
+		if (att_op_get_type(hdr->code) != ATT_COMMAND &&
```

Comment:
This patch is not mentioned anywhere, is it here intentionally?

Reply:
I was asked to cherry-pick a [fromtree] by @MarekPieta. Given that this PR didn't make it for the 1.8 release I assume it will be cherry-picked elsewhere, so I can drop it.

Reply:
Anyway, this will be added in a separate PR, so dropped it from this PR :)